package.json

```diff
@@ -4,14 +4,14 @@
   "private": true,
   "workspaces": [
     "sandpack-client",
-    "sandpack-react",
-    "sandpack-playground"
+    "sandpack-react"
   ],
   "description": "",
   "scripts": {
     "lint": "eslint '**/*.ts?(x)' --fix",
     "format": "prettier --write '**/*.ts?(x)'",
     "build": "yarn workspaces run build",
+    "build:publish": "yarn workspaces run build:publish",
```

Comment:
can you document how the publish process works in the Readme file? I know we talked about the semantic release a while ago, but it make sense for the flow to be described for everyone that wants to contribute on the long term